### PR TITLE
is_public field in Phase model created.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#364 solved with migrations added. @deshraj Please review the PR.

### CODE REVIEW

**Comment by deshraj** [comment]
@RishabhJain2018 please modify the tests related to the phases.

**Comment by RishabhJain2018** [comment]
@deshraj okk sure.

**Comment by deshraj** [comment]
@RishabhJain2018 see this https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/serializers.py#L38. You are missing something here.

**Comment by RishabhJain2018** [comment]
@deshraj Please review the PR.

**Comment by deshraj** [comment]
Looks good. :-)

File: apps/challenges/migrations/0010_merge_20170108_0823.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```
Comment (TroJan) [inline apps/challenges/migrations/0010_merge_20170108_0823.py]: @RishabhJain2018 , @deshraj : can you please explain me why is this file here??

Comment (deshraj) [inline apps/challenges/migrations/0010_merge_20170108_0823.py]: Oh I see, my bad. I think I missed this and forgot to ask Rishabh to just modify the previous migration file.
